Sources/Widgets/Widgets3D/ResliceCursorWidget/helpers.js

```diff
@@ -309,3 +275,67 @@ export function transformPlane(planeToTransform, targetPlane, viewType) {
   planeToTransform.setPoint1(rotatedPoint1);
   planeToTransform.setPoint2(rotatedPoint2);
 }
+
+// Get name of the line in the same plane as the input
+export function getAssociatedLinesName(lineName) {
+  switch (lineName) {
+    case 'AxisXinY':
+      return 'AxisZinY';
+    case 'AxisXinZ':
+      return 'AxisYinZ';
+    case 'AxisYinX':
+      return 'AxisZinX';
+    case 'AxisYinZ':
+      return 'AxisXinZ';
+    case 'AxisZinX':
+      return 'AxisYinX';
+    case 'AxisZinY':
+      return 'AxisXinY';
+    default:
+      return '';
+  }
+}
+
+/**
+ * Get the line name, constructs from the plane name and where the plane is displayed
+ * Example: planeName='X' rotatedPlaneName='Y', then the return values will be 'AxisXinY'
+ * @param {String} planeName Value between 'X', 'Y' and 'Z'
+ * @param {String} rotatedPlaneName Value between 'X', 'Y' and 'Z'
+ * @returns {String}
+ */
+export function getLineNameFromPlaneAndRotatedPlaneName(
+  planeName,
+  rotatedPlaneName
+) {
+  return `Axis${planeName}in${rotatedPlaneName}`;
+}
+
+/**
+ * Extract the plane name from the line name
+ * Example: 'AxisXinY' will return 'X'
+ * @param {String} lineName Should be following this template : 'Axis_in_' with _ a character
+ * @returns {String} Value between 'X', 'Y' and 'Z'
+ */
+export function getPlaneNameFromLineName(lineName) {
+  const match = lineName.match('([XYZ])in[XYZ]');
+  if (match) {
+    return match[1];
+  }
+  return '';
```

Comment:
I think returning `undefined` or `null` is more appropriate. It carries a message that a problem occured